app/assets/stylesheets/views/_browse.scss

```diff
@@ -1,11 +1,18 @@
 @import "govuk_publishing_components/individual_component_support";
 
+// Dart Sass 1.79.0 introduced support for CSS Color Level 4 color spaces.
+// Consequently, the `govuk-tint` mixin now outputs rgb values with floating-point precision,
+// leading to unsupported property value warnings in some browsers, such as Safari 11.1.1.
+// To maintain compatibility, we're defining equivalent hex color values as variables.
+
+$govuk-blue-tint-80: #d2e2f1; // govuk-tint(govuk-colour("blue"), 80%)
```

Comment:
Just wondering if this section is needed here, it looks like the `$govuk-blue-tint-80` variable will be available through the `individual_component_support` import